index.html

```diff
@@ -16,20 +16,38 @@
     <link rel="stylesheet" href="assets/css/app.css">
 </head>
 <body>
-<header>
-    <div class="navbar bg-white">
-        <div class="container">
-            <a id="" href="#" class="navbar-welcome d-flex align-center">
-                <strong>Welcome</strong>
-            </a>
-            <nav class="navbar-menu">
-                <span class="navbar-menu-icon"></span>
-            </nav>
+<main id="main">
+    <header>
+        <div class="navbar">
```

Comment:
- [ ] The background image works fine at some screen sizes, but at some screen sizes it doesn't get centered properly and also blocks the nav links view and gives an impression to the user that the elements of the hero section is not centered properly which they are and you done a remarkable job there. kindly consider fixing the background image and making it responsive to work fine at all screens.
your design
![qwe](https://user-images.githubusercontent.com/96683843/214649012-bda567e7-59c7-4828-9af1-b95b1e9dd4dd.jpg)
- [ ] Also the background image at the mobile screen at less than `768px`, gets fixed width and aligned to the left of the screen which also is not a good user experience. kindly consider making it responsive by not giving it fixed width.
![tr](https://user-images.githubusercontent.com/96683843/214649722-2e94f5f0-1e0d-40bb-9fff-d8a979e9e769.jpg)